### ISSUE TITLE
[Xamarin-Profiler] Accessibility: MAS2.1.1: OSX: Keyboard: The Sorting buttons for table's header rows, of " Instrument Detail Area", are not accessible using keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#453]
### Steps to Reproduce

Pre-Requisite: 

1. Install Xamarin Studio.
2. Launch Xamarin Studio.
3. Navigate to top menu bar of the Xamarin studio> File> New Solution> Press ENTER
("Choose a template for your new project" wizard is opened)
4.  Navigate to App under Multiplatform> Forms app under Xamarin.Forms
5. Navigate to Next button> Press ENTER
('Configure your Form app' wizard is opened)
6. Fill the App name, check all the target platform-check box, select use portable class library radio button, use XAML for user interface files.
7. Navigate to Next button> Press ENTER
('Configure your new project" wizard is opened)
8. Navigate to Create> Press ENTER.
(New project is created)

Repro Steps:
"Scenario 1:
1. Run Pre-requisite
2. Navigate to Top status bar of ""Xamarin Studio"" > Run> Start Profiling, then Press ENTER to launch Xamarin Profiler.
 (Xamarine Profiler is launched.)
3. Navigate to ""General"" Side bar (LHS)
4. Navigate to ""Allocations/All instruments"" 
5. Navigate the ""Start Profiling"" buttons at the bottom of the wizard, Press ENTER.
(Profiling started now)
6. Navigate to > ""Allocations"" Side-bar> ""Allocations"" Tab of "" Instrument Detail Area"". 
7. Navigate to ""Size"" header row of the table.
8. Try to navigate the ""Size's ascending/descending order button"" to switch/change into ascending/descending order for size coloumn.

Scenario 2:
1. Run Pre-requisite
2. Navigate to Top status bar of ""Xamarin Studio"" > Run> Start Profiling, then Press ENTER to launch Xamarin Profiler.
 (Xamarine Profiler is launched.)
3. Navigate to ""General"" Side bar (LHS)
4. Navigate to ""Allocations/All instruments"" 
5. Navigate the ""Start Profiling"" buttons at the bottom of the wizard, Press ENTER.
(Profiling started now)
6. Navigate to > ""Allocations"" Side-bar> ""Call Tree"" Tab of "" Instrument Detail Area"" . 
7. Navigate to ""Byte Used"" header row of the table.
8. Try to navigate the ""Byte Used'' ascending/descending order button"" to switch/change into ascending/descending order for size coloumn.

Scenario 3:
1. Run Pre-requisite
2. Navigate to Top status bar of ""Xamarin Studio"" > Run> Start Profiling, then Press ENTER to launch Xamarin Profiler.
 (Xamarine Profiler is launched.)
3. Navigate to ""General"" Side bar (LHS)
4. Navigate to ""Allocations/All instruments"" 
5. Navigate the ""Start Profiling"" buttons at the bottom of the wizard, Press ENTER.
(Profiling started now)
6. Navigate to > ""Allocations"" Side-bar> ""Call Tree"" Tab of "" Instrument Detail Area"". 
7. Navigate to ""Symbol name"" column of the table.
8. Try to navigate the ""expand/collapse'  button"" to switch.

### Expected Behavior
"Scenario 1:
The ""Size's ascending/descending order button"" of "Instrument Detail Area", should be accessible using keyboard.

Scenario 2:
The ""Byte Used's ascending/descending order button"" of "" Instrument Detail Area"", should be accessible using keyboard.

Scenario 3:
The ""Expand/Collapse button"" under symbol name column, of "" Instrument Detail Area"", should be accessible using keyboard."
### Actual Behavior
"Scenario 1:
The ""Size's ascending/descending order button"" of "Instrument Detail Area", is not accessible using keyboard.

Scenario 2:
The ""Byte Used's ascending/descending order button"" of "" Instrument Detail Area"", is not accessible using keyboard.

Scenario 3:
The ""Expand/Collapse button""  under symbol name column of "" Instrument Detail Area"", is not accessible using keyboard."
### User impact:
The user who completely rely on keyboard, won't able to change to ascending/descending order.

![sorting button_byteused_notaccessible with keyboard](https://user-images.githubusercontent.com/22956130/41768188-2470ad70-7629-11e8-904d-2047c5812170.png)
![sorting button_size_notaccessible with keyboard](https://user-images.githubusercontent.com/22956130/41768204-28fe8272-7629-11e8-807a-6be8f50e2519.png)
![sorting button_symbolname_expandcollapsebutton_notaccessible with keyboard](https://user-images.githubusercontent.com/22956130/41768212-2ccb2d24-7629-11e8-9bea-62c630d8ab0a.png)

### Environment
Testing Environment:
OS: OSX (Version: 10.11.6)
Xamarin-Profiler: 1.0.4

**Comment by WiproPSAccTest** [comment]
Please add the following labels
A11y15.8ExceptionRequested
A11yMAS
Accessibility
MAS2.1.1
Xamarin-Profiler
Wipro
